.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Global Reviewers
+# review when someone opens a pull request.
+*       @cartyc @fmichaelobrien
```

Comment:
Thanks Chris, fmichaelobrien is google only (primary), obriensystems is only the backup